test/WufooTranslator_test.js

```diff
@@ -160,23 +159,24 @@ describe('WufooTranslator', function() {
           'DateUpdated': '2014-12-03 17:51:07',
           'Hash': '0123456789abcd'
         }),
-        Field106: "Fred",
-        Field107: "Flinstone",
-        Field114: "fred@flinstone.com",
-        Field108: "301 Cobblestone Way",
-        Field109: "",
-        Field110: "Bedrock",
-        Field111: "South Dakota",
-        Field112: "12345",
-        Field113: "United States",
+        Field106: 'Fred',
+        Field107: 'Flinstone',
+        Field114: 'fred@flinstone.com',
+        Field108: '301 Cobblestone Way',
+        Field109: '',
+        Field110: 'Bedrock',
+        Field111: 'South Dakota',
+        Field112: '12345',
+        Field113: 'United States',
         Field4: '555-555-1212',
         Field5: '543-210-1234',
-        CreatedBy: "public",
-        DateCreated: "2014-12-03 18:07:46",
-        EntryId: "6",
-        IP: "127.0.0.1",
-        HandshakeKey: ""
-      }, function(err, sub) {
+        CreatedBy: 'public',
+        DateCreated: '2014-12-03 18:07:46',
+        EntryId: '6',
+        IP: '127.0.0.1',
+        HandshakeKey: ''
+      }, function (err, sub) {
+        should(err).not.exist
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.